File: cpp/program/ProdukImpor.cpp
#include "ProdukElektronik.cpp"
#include <iostream>
using namespace std;

class ProdukImpor : public ProdukElektronik {
private:
    string negara;
    string nomorAPI;
    string invoice;

public:
    ProdukImpor(string id = "", string n = "", string d = "", double h = 0,
                string m = "", string mdl = "", string t = "",
                string neg = "", string api = "", string inv = "")
        : ProdukElektronik(id, n, d, h, m, mdl, t),
          negara(neg), nomorAPI(api), invoice(inv) {}

    string getNegara() const { return negara; }
    string getNomorAPI() const { return nomorAPI; }
    string getInvoice() const { return invoice; }

    static ProdukImpor tambah() {
        string id, n, d, m, mdl, t, neg, api, inv;
        double h;

        cout << "\n=== Input Data Produk Impor ===\n";
        cout << "ID Produk   : "; getline(cin, id);
        cout << "Nama        : "; getline(cin, n);
        cout << "Deskripsi   : "; getline(cin, d);
        cout << "Harga       : "; cin >> h; cin.ignore();
        cout << "Merek       : "; getline(cin, m);
        cout << "Model       : "; getline(cin, mdl);
        cout << "Tegangan    : "; getline(cin, t);
        cout << "Negara      : "; getline(cin, neg);
        cout << "Nomor API   : "; getline(cin, api);
        cout << "Invoice     : "; getline(cin, inv);

        return ProdukImpor(id, n, d, h, m, mdl, t, neg, api, inv);
    }
};

File: php/program/ProdukImpor.php
<?php
require_once "ProdukElektronik.php";

class ProdukImpor extends ProdukElektronik {
    private $negara;
    private $nomorAPI;
    private $invoice;
    private $gambar;

    public function __construct($id = "", $nama = "", $deskripsi = "", $harga = 0,
                                $merek = "", $model = "", $tegangan = "",
                                $negara = "", $api = "", $invoice = "", $gambar = "") {
        parent::__construct($id, $nama, $deskripsi, $harga, $merek, $model, $tegangan);
        $this->negara = $negara;
        $this->nomorAPI = $api;
        $this->invoice = $invoice;
        $this->gambar = $gambar;
    }

    public function getNegara() { return $this->negara; }
    public function getNomorAPI() { return $this->nomorAPI; }
    public function getInvoice() { return $this->invoice; }
    public function getGambar() { return $this->gambar; }
}
